includes/REST_API/Stories_Controller.php

```diff
@@ -175,14 +137,8 @@ public function get_item_schema() {
 		if ( $this->schema ) {
 			return $this->add_additional_fields_schema( $this->schema );
 		}
-		$schema = parent::get_item_schema();
 
-		$schema['properties']['story_data'] = [
-			'description' => __( 'Story data stored as a JSON object. Stored in post_content_filtered field.', 'web-stories' ),
-			'type'        => 'object',
-			'context'     => [ 'edit' ],
-			'default'     => [],
-		];
+		$schema = parent::get_item_schema();
 
 		$schema['properties']['featured_media_url'] = [
```

Comment:
Why are `featured_media_url`, `publisher_logo_url`, `style_presets` not on template post type? Shouldn't these be the same?